### DISCUSSION TITLE
Ollama Load balancing

### DISCUSSION THREAD

**Discussion by tedraymond** [open-webui/open-webui#2526] [General]
Curious how the backend chooses which ollama instance to use.

I have a couple remote ollama instances and it seems to choose which to use at random times

**Comment by justinh-rahb** [comment]
> random

☝️

**Reply by tedraymond** [reply]
I ran into a weird one on this. It seems to be random but also seems to prioritize the last Ollama host defined. I tested today by shutting down the last host, and the web UI would take some time to completely load, upwards of 30 seconds. But would eventually load and use the next to last host, total of 4 hosts defined.

When the UI would respond I can delete the missing host from the connection list and the UI would act normal.

Curious if anyone else has seen this.
